src/scheduler/MpiWorld.cpp

```diff
@@ -403,6 +389,55 @@ void MpiWorld::send(int sendRank,
     }
 }
 
+void MpiWorld::recv(int sendRank,
```

Comment:
Moved `recv` here to have `isend`, `irecv`, `send`, and `recv`'s implementations together, noisy diff however.